darts/models/forecasting/torch_forecasting_model.py

```diff
@@ -2044,7 +2043,7 @@ def _optimized_historical_forecasts(
         last_points_only: bool = True,
         verbose: bool = False,
         show_warnings: bool = True,
-        predict_likelihood_parameters: bool = False,
+        **kwargs,
```

Comment:
The signature here is now different from ForecastingModel._optimized_historical_forecasts and RegressionModel._opt....

Either leave or drop `num_samples` and `predict_likelihood_parameters` in all of them